modules/460-log-shipper/images/vector/Dockerfile

```diff
@@ -20,14 +33,29 @@ RUN cargo build \
     -j $(($(nproc) /2)) \
     --offline \
     --no-default-features \
-    --features "api,api-client,enrichment-tables,sources-host_metrics,sources-internal_metrics,sources-file,sources-kubernetes_logs,transforms,sinks-prometheus,sinks-blackhole,sinks-elasticsearch,sinks-file,sinks-loki,sinks-socket,sinks-console,sinks-vector,sinks-kafka,sinks-splunk_hec,unix,rdkafka?/gssapi-vendored,vrl-cli" \
+    --features "api,api-client,enrichment-tables,sources-host_metrics,sources-internal_metrics,sources-file,sources-kubernetes_logs,transforms,sinks-prometheus,sinks-blackhole,sinks-elasticsearch,sinks-file,sinks-loki,sinks-socket,sinks-console,sinks-vector,sinks-kafka,sinks-splunk_hec,unix,rdkafka?/dynamic-linking,rdkafka?/gssapi-vendored,vrl-cli" \
     && strip target/release/vector
 
+
+### 2: Final image
 FROM $BASE_UBUNTU
 RUN mkdir -p /etc/vector \
     && apt-get update \
-    && apt-get install -yq ca-certificates tzdata inotify-tools gettext procps \
+    && apt-get install -yq ca-certificates tzdata inotify-tools gettext procps wget \
     && rm -rf /var/cache/apt/archives/*
-COPY --from=build /vector/vector/target/release/vector /usr/bin/vector
+
+# libssl.1
+RUN wget http://nz2.archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.16_amd64.deb \
+    && dpkg -i libssl1.1_1.1.1f-1ubuntu2.16_amd64.deb
+# libcurl.4
+RUN apt-get install -yq libcurl4
+
+COPY --from=build-vector /vector/vector/target/release/vector /usr/bin/vector
+
+COPY --from=build-vector /usr/local/lib/librdkafka.* /usr/local/lib/
+COPY --from=build-vector /usr/local/lib/pkgconfig/rdkafka.pc /usr/local/lib/pkgconfig/rdkafka.pc
+
+ENV LD_LIBRARY_PATH=/usr/local/lib
```

Comment:
Seems redundant, `/usr/local/lib` is configured to search in Ubuntu.

Reply:
Nah, I don't know exactly why, but without this variable, the code does not work. According to the `/etc/ld.so.conf` everything is configured appropriately, but I can see only errors without this env variable.